app/views/devise/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.